ledger-core/conveyor/conveyor_semaphore.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package conveyor
+
+import (
+	"runtime"
+
+	"github.com/insolar/assured-ledger/ledger-core/conveyor/smachine"
+	"github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/smsync"
+)
+
+type ParallelProcessingLimiter interface {
```

Comment:
do you really need an interface? 
as you can't mock SemaChildLink anyway, this is futile